templates/base/src/utils/Logger/index.js

```diff
@@ -1 +1,7 @@
-export { default } from './logger';
+import { default as createLogger } from './log-util';
+import { default as createDefaultLogger } from './logger';
+
+export {
+    createLogger,
+    createDefaultLogger
```

Comment:
There is no much value in exporting createDefaultLogger, as this one can be initialised when someone is initialising createLogger.